test/evp_pkey_provided_test.c

```diff
@@ -128,6 +128,11 @@ static int compare_with_file(const char *alg, int type, BIO *membio)
     return ret;
 }
 
+static int pass_cb(char *buf, int size, int rwflag, void *u)
+{
+    return 0;
```

Comment:
and I wonder if a negative test, where the callback return -1 would make sense too.

Reply:
Yes, that would make sense to add.

Reply:
@bernd-edlinger added the test

Reply:
excellent!